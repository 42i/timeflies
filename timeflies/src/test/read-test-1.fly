
wp    aaa   50
	bbb
		bbbb
	ccc
		cccc
		
wp aaa.bbb
	bbbb2 60 , green leaves on the tree
	
day 2012-07-18, hours 8  18 ,   off 1
- aaa.bbb   3.5 , did some mucking about