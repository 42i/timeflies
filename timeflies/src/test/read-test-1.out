Work package breakdown:
ALL
    aaa
        bbb
            bbbb
            bbbb2 -- green leaves on the tree
        ccc
            cccc
Time at work overview (2012-07):
2012-07-18, Wed:  9.00 worked, --.-- leave, --.-- ill
weekly: must =   8.00, worked =   9.00, ill =   0.00, leave taken=   0.00, leave left =   0.00, have =   9.00
global: must =   8.00, worked =   9.00, ill =   0.00, leave taken=   0.00, leave left =   0.00, have =   9.00
Work package summary (2012-07):
  3.50 : ALL
      3.50 : aaa
          3.50 : bbb
